src/tools/miri/src/shims/foreign_items.rs

```diff
@@ -731,6 +731,10 @@ trait EvalContextExtPriv<'tcx>: crate::MiriInterpCxExt<'tcx> {
                 this.mem_copy(ptr_src, ptr_dest, Size::from_bytes(n), true)?;
                 this.write_pointer(ptr_dest, dest)?;
             }
+            "__rust_no_alloc_shim_is_unstable_v2" => {
```

Comment:
You put this in the section "C memory handling functions". That doesn't seem right.

Putting it with the "Rust allocation" seems best.